### ISSUE TITLE
Failed to deploy inception

### ISSUE THREAD

**Issue by o-orand** [orange-cloudfoundry/paas-templates#1647]
### Expected behavior

* As a paas-templates operator
* In order to upgrade paas-templates
* I need to have an up and running inception deployment

### Observed behavior
Failed to process inception manifest with: 
```
Error 'operation [1] in ./bosh-inputs/operators/vsphere-operators.yml failed': Expected to find exactly one matching array item for path '/instance_groups/name=rescue' but found 0
``` 

Extract from `micro-depls/inception/template/vsphere/vsphere-operators.yml`
```
#--- Set routes for internal access
- type: replace
  path: /instance_groups/name=rescue/jobs/-
```

But there is only `inception` instance_groups defined is manifest

### Affected releases

* 54.0.0

**Comment by o-orand** [comment]
First issue solve, but we still fail ti deploy inception due to: 
```
Task 11700503 | 12:08:35 | Error: - Failed to find variable '/micro-bosh/inception/intranet_interco_inception_ip' from config server: HTTP Code '404', Error: 'The request could not be completed ...'
```

**Comment by ogrand** [comment]
Fix in feature-fix-54.0.1 branch
